go/vt/vtctl/vtctl.go

```diff
@@ -2802,7 +2802,7 @@ func commandValidateSchemaKeyspace(ctx context.Context, wr *wrangler.Wrangler, s
 	if *excludeTables != "" {
 		excludeTableArray = strings.Split(*excludeTables, ",")
 	}
-	return wr.ValidateSchemaKeyspace(ctx, keyspace, excludeTableArray, *includeViews, *skipNoMaster)
+	return wr.ValidateSchemaKeyspace(ctx, keyspace, excludeTableArray, *includeViews, *skipNoMaster, false /*includeVSchema*/)
```

Comment:
We should add a flag to this command (and probably `ValidateSchemaShard`, sorry for not pointing that out earlier) to be able to pass `true` through here.